src/platform/forms/constants.js

```diff
@@ -282,3 +282,235 @@ export const SIP_ENABLED_FORMS = new Set([
   VA_FORM_IDS.FORM_VA_2346A,
   VA_FORM_IDS.FORM_5655,
 ]);
+
```

Comment:
We may want to add a "disclaimer" here in comments in case teams are updating Form configs.  Maybe just something to direct them to existing docs while we work on this, and that this array is work in progress for testing 🤔